images/chaos-daemon/Dockerfile

```diff
@@ -27,43 +42,13 @@ RUN case "$TARGET_PLATFORM" in \
     ;; \
     *) echo >&2 "error: unsupported architecture '$TARGET_PLATFORM'"; exit 1 ;; \
     esac; \
-    curl -k -L https://github.com/chaos-mesh/nsexec/releases/download/v0.1.6/nsexec-$NSEXEC_ARCH-unknown-linux-gnu.tar.gz | tar xz -C /tmp/bin; \
-    curl -k -L https://github.com/chaos-mesh/chaos-tproxy/releases/download/v0.5.3/tproxy-$NSEXEC_ARCH.tar.gz | tar xz -C /tmp/bin; \
-    curl -k -L https://github.com/chaos-mesh/memStress/releases/download/v0.3/memStress_v0.3-$NSEXEC_ARCH-linux-gnu.tar.gz | tar xz -C /tmp/bin
-
-
-
-
-# ---
-FROM debian:bullseye-slim
-
-ARG HTTPS_PROXY
-ARG HTTP_PROXY
-
-ENV http_proxy $HTTP_PROXY
-ENV https_proxy $HTTPS_PROXY
-
-RUN mkdir -p /usr/share/man/man1 /usr/share/man/man2
-
-RUN apt-get update && \
-    echo "deb http://security.debian.org/debian-security bullseye-security main contrib non-free" >> /etc/apt/sources.list && \
-    echo "deb http://deb.debian.org/debian bullseye-proposed-updates main contrib non-free" >> /etc/apt/sources.list && \
-    apt --allow-releaseinfo-change update && apt upgrade  -y && \
-    apt-get install -y tzdata iptables ebtables net-tools ipset stress-ng iproute2 fuse util-linux procps openjdk-11-jre && \
-    rm -rf /var/lib/apt/lists/*
-
-RUN update-alternatives --set iptables /usr/sbin/iptables-legacy && \
-    update-alternatives --set ebtables /usr/sbin/ebtables-legacy
-
-ENV RUST_BACKTRACE 1
-ENV BYTEMAN_HOME /usr/local/byteman
-ENV PATH $PATH:/usr/local/byteman/bin
-ENV JAVA_HOME /usr/lib/jvm/java-11-openjdk-amd64
-ENV PATH $PATH:/usr/local/bin
+    curl -L https://github.com/form3tech-oss/toda/releases/download/v0.2.3-f3-rorootfs-pre1/toda-linux-amd64.tar.gz | tar xz -C /urs/local/bin; \
+    curl -L https://github.com/chaos-mesh/nsexec/releases/download/v0.1.6/nsexec-$NSEXEC_ARCH-unknown-linux-gnu.tar.gz | tar xz -C /usr/local/bin; \
+    curl -L https://github.com/chaos-mesh/chaos-tproxy/releases/download/v0.5.3/tproxy-$NSEXEC_ARCH.tar.gz | tar xz -C /usr/local/bin; \
+    curl -L https://github.com/chaos-mesh/memStress/releases/download/v0.3/memStress_v0.3-$NSEXEC_ARCH-linux-gnu.tar.gz | tar xz -C /usr/local/bin
```

Comment:
Can we add ENV vars for `TODA_VERSION`, `NSEXEC_VERSION`, `TPROXY_VERSION` and `MEMSTRESS_VERSION` ?

Reply:
Sure